zul/src/archive/web/js/zul/sel/Listbox.js

```diff
@@ -475,6 +475,12 @@ zul.sel.Listbox = zk.$extends(zul.sel.SelectWidget, {
 				}
 			}
 		}
+	},
+	getFirstItemIndex: function () {
```

Comment:
This is a new api, If you don't want to maintain it , you can write _ getFirstItemIndex. (voluntary)